configs/projects/_shared-pe-bolt-server_with_ruby.rb

```diff
@@ -176,6 +176,7 @@
 proj.component('rubygem-net-ssh')
 proj.component('rubygem-net-ssh-krb')
 proj.component('rubygem-nori')
+proj.component('rubygem-rubyntlm')
```

Comment:
Oh, when I reverted the usage of the ntlm fork in pe-installer-runtime-main (d9934091496a68bb4bdf3f1b07ce67cc65d852ff) I didn't see a dependency on 'rubygem-rubyntlm' added. Does something similar need to be done to pe-instlaler?

Reply:
Not now, when i was looking at this i realized for installer we actually use rubygems to do dependency resolution. This is not ideal because https://tickets.puppetlabs.com/browse/PE-36494 We will explicitly package that when we do that ticket but for now rubygems will find the latest rubyntmlm and we will be good to go.